### PR TITLE
YCheckBoxFrame disable as initial status wrong behavior 

### PR DESCRIPTION
**Pull request by anaselli**
This commit fix issue #81.

### CODE REVIEW

File: src/YGFrame.cc [hunk 1]
```diff
@@ -134,6 +134,8 @@ class YGCheckBoxFrame : public YCheckBoxFrame, public YGBaseFrame
 
 		setLabel (label);
 		setValue (checked);
+    doSetEnabled( checked );
```
Comment (shundhammer) [inline src/YGFrame.cc:137]: Please notice that this behaviour should be inverted if the `invertAutoEnable()` flag is set (see https://github.com/libyui/libyui/blob/master/libyui/src/YCheckBoxFrame.h#L105).

So AFAICS this should rather be

```C++
doSetEnabled( checked && ! invertAutoEnable() )
```

(Please check if this is correct)

Comment (anaselli) [reply src/YGFrame.cc:137]: Interesting, i just changed the code and didn't know about that property. Thanks for the hint, I'll do some test more following your suggestion :) Thanks, will call you again after.

Comment (anaselli) [reply src/YGFrame.cc:137]: Sorry for the delay. I've experienced that that property does not work properly at start-up with your suggestion. I'm investigating. BTW i've exprienced it also using Qt. I will provide e simple test example and open a related issue if after testing again that is confirmed

Comment (anaselli) [reply src/YGFrame.cc:137]: @shundhammer i've fixed the autoenable/inverautoenable behavior see https://github.com/libyui/libyui /issues/45 and issue #81

**Comment by anaselli** [comment]
I have a little spare time, but i cannot fix any libyui changes if my work requires months and approving more months :p  

BTW If you're interested i tried a qt patch for the behaviour of invertAutoEnable but i haven't committed it, i could give you what i did eventually...

**Review by shundhammer — APPROVED** [review]
LGTM (but I don't have any Gtk experience)

**Pull request merged**